cmd/createcluster.go

```diff
@@ -120,13 +120,17 @@ func bindInsecureFlags(flags *pflag.FlagSet, insecureKeys *bool) {
 
 func runCreateCluster(ctx context.Context, w io.Writer, conf clusterConfig) error {
 	var err error
-	if conf.Clean {
-		// Remove previous directories
-		if err = os.RemoveAll(conf.ClusterDir); err != nil {
-			return errors.Wrap(err, "remove cluster dir")
-		}
-	} else if _, err = os.Stat(path.Join(nodeDir(conf.ClusterDir, 0), "cluster-lock.json")); err == nil {
-		return errors.New("existing cluster found. Try again with --clean")
+
+	if conf.NumNodes == 0 {
```

Comment:
we can also use `wrapPreRunE` function to do these config validations like we are using in other commands, WDYT?

Reply:
Important thing is that this is tested.

Don't mind adding this in the command itself, what about extracting a `verifyConfig` and calling that here?

Reply:
Yeah I can extract config checks in verifyConfig.

> Important thing is that this is tested.

It is, I added test cases specifically to test configuration verification.